packages/synthetics-sdk-broken-links/src/broken_links.ts

```diff
@@ -33,39 +33,97 @@ import {
 export async function runBrokenLinks(
   options: BrokenLinksResultV1_BrokenLinkCheckerOptions
```

Comment:
if the caller (typically a js file) calls this without options ie `runBrokenLinks()`, what happens?

Reply:
Ok can you add a todo?

Reply:
done.